src/routes/team/[team]/(teamPages)/buckets/+page.svelte

```diff
@@ -103,6 +105,20 @@
 			}}
 		/>
 	</Card>
+	<Card>
+		<h3>CDN buckets</h3>
+		<p>
+			All teams have their own team buckets, you can administer these with <em>gcloud storage</em>
```

Comment:
Burde vi ha lenke til doc?
https://cloud.google.com/sdk/gcloud/reference/storage

Reply:
A link to our own docs on the topic